corehq/apps/events/models.py

```diff
@@ -55,14 +55,19 @@ class AttendanceTrackingConfig(models.Model):
         default=DEFAULT_ATTENDEE_CASE_TYPE,
     )
 
+    @staticmethod
     def toggle_mobile_worker_attendees(domain, value):
         config, _created = AttendanceTrackingConfig.objects.get_or_create(domain=domain)
         config.mobile_worker_attendees = value
         config.save()
 
+    @staticmethod
     def mobile_workers_can_be_attendees(domain):
-        config, _created = AttendanceTrackingConfig.objects.get_or_create(domain=domain)
-        return config.mobile_worker_attendees
+        try:
```

Comment:
I made this change because the way it previously was would have created an `AttendanceTrackingConfig` record simply by calling `mobile_workers_can_be_attendees`. Since we call this when creating a mobile worker - which can happen on a plan not supporting Attendance Tracking - this would have created a non-privilged record of `AttendanceTrackingConfig`